db/schema.rb

```diff
@@ -44,8 +44,8 @@
   end
 
   create_table "api_users", force: :cascade do |t|
-    t.string "name", null: false
```

Comment:
Was this not updated from a previous migration?

Reply:
When I ran the migrations I got a few unrelated changes to the schema, I'm not sure where from specifically. I'll remove them from this PR.